paper/paper.md

```diff
@@ -0,0 +1,469 @@
+We are bioinformatics trainees at the University of Michigan who started a
+local chapter of Girls Who Code to provide a fun and supportive environment for
+high school women to learn the power of coding.
+Our goal was to cover basic coding topics and data science concepts through
+live-coding and hands-on practice.
+However, we couldn’t find a resource that exactly met our needs. Therefore, over the past
+three years, we have developed a curriculum and instructional format using
+Jupyter notebooks to effectively teach introductory Python for data science.
+This method, inspired by The Carpentries organization, uses bite-sized lessons
+followed by independent practice time to reinforce coding concepts, and
+culminates in a data science capstone project using real-world data.
+We believe our open curriculum is a valuable resource to the wider education
+community and hope that educators will use and improve our lessons, practice
+problems, and teaching best practices.
+Anyone can contribute to [our educational materials on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women bioinformatics trainees at the
+University of Michigan, we experience the gender gap in our field first-hand.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We also see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level; during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program was
+35.2%, and the average percentage of female applicants listing Bioinformatics as first or second
+choice in the Program in Biomedical Sciences, U-M's biomedical PhD umbrella
+program was 41%.
+
+Previous research on women's educational experiences in science, technology, engineering,
+and mathematics (STEM) have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+Additionally, women often experience stereotype threats that
+influence their math and science performance as well as their career goals [@hill_why_2010].
+The majority of our organization's founding graduate students (all women) began
+coding in our undergraduate careers or later.
+We wanted to provide a safe place for local young high school women to
+develop confidence in themselves and their computational skills
+before college, and be exposed to successful female role models in STEM to counter negative
+stereotypes.
```

Comment:
Would this be a good place to explicitly mention bringing in guest speakers?

Reply:
I feel like since this is in the intro and we don't talk about our curriculum until later, it doesn't really fit here. We mention it below in Experience of Use (lines 373-379). But maybe we should add a description of a typical day somewhere to the curriculum section?

Reply:
@hayse1 does that make sense to you? Anywhere else you think we should include it?

Reply:
I think it's fine!